src/Ranking.elm

```diff
@@ -1,17 +1,105 @@
 module Ranking exposing (Model, Msg, init, update, view)
 
-import Debug exposing (todo)
-import Zipper exposing (Zipper)
+import Browser
+import List.Zipper exposing (current, after)
+import SetZipper exposing (crateZipper)
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (centerX, centerY, column, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Maybe
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view >> layout []
+        , update = update
+        }
 
 
 type alias Model =
     { teams : Zipper (Zipper Team)
-    , isShowingInfo : Bool
+    , isShowingInfo : Bool -- What I need to do with this variable?
```

Comment:
If I recall correctly, this is for knowing if the current team has an expanded info about it. The table only shows the team's number, name, rank, etc. But you have the option to click on a team and expand it's row in the table to show more info about it.
If you at the end you conclude it is not necessary, you can remove it.